chained-actions/assets

```diff
@@ -0,0 +1 @@
+../assets
```

Comment:
We might not need this if we just call configure_owsk.sh directly as I indicated in my other comments in index.json

Reply:
that script comes from the shared assets directory.  That directory doesn't get bundled in to a scenario without the symlink from what I've been able to determine.  Would be super thrilled to be educated otherwise.